pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -553,19 +560,19 @@ func (d *namespacedResourcesDeleter) deleteAllContent(ns *v1.Namespace) (int64,
 	}
 
 	// if len(errs)==0, NewAggregate returns nil.
-	klog.V(4).Infof("namespace controller - deleteAllContent - namespace: %s, estimate: %v, errors: %v", namespace, estimate, utilerrors.NewAggregate(errs))
+	logger.V(4).Info("namespace controller - deleteAllContent", "nameSpace", namespace, "estimate", estimate, "err", utilerrors.NewAggregate(errs))
```

Comment:
You can add `err := utilerrors.NewAggregate(errs)` before this and then use `err` here and in the return in the next line.

Reply:
done